_resources/_styles/type.styl

```diff
@@ -34,6 +34,10 @@ h1,h2,h3,h4,h5,h6 {
 	position relative
 	cursor: default;
 }
+h1 {
```

Comment:
Why apply a different style here than `.Display1`? Keep styling under the class and leave the browser defaults alone for now.